tests/ttnn/unit_tests/operations/test_moreh_clip_grad_norm.py

```diff
@@ -88,66 +67,70 @@ def test_moreh_clip_grad_norm(
             param.grad = grad
 
             cpu_inputs.append(param)
```

Comment:
Can refactor to torch/ttnn but doesn't matter too much

Reply:
I think this works well enough.